products/mssqlserver.md

```diff
@@ -86,13 +94,58 @@ releases:
 
 -   releaseCycle: "2005"
     releaseLabel: "2005 SP4"
+    codename: Yukon
     releaseDate: 2006-01-14
     support: 2011-04-12
     eol: 2016-04-12
     extendedSupport: false
     latest: "9.0.5324.0"
     link: https://support.microsoft.com/help/2716427
 
+-   releaseCycle: "2000"
+    releaseLabel: "2000 SP4"
+    codename: Shiloh
+    releaseDate: 2000-11-30
+    support: 2008-04-08
+    eol: 2013-04-09
+    extendedSupport: false
+    latest: "8.0.2305"
+    latestReleaseDate: 2012-08-14
+    link: https://support.microsoft.com/help/983811
+
+-   releaseCycle: "7.0"
+    releaseLabel: "7.0 SP4"
+    codename: Sphinx
+    releaseDate: 1998-11-27
+    support: 2005-12-31
+    eol: 2011-01-11
+    extendedSupport: false
+    latest: "7.0.1152"
+    latestReleaseDate: 2012-05-09
+    link: https://support.microsoft.com/help/948113
+
+-   releaseCycle: "6.5"
+    releaseLabel: "6.5 SP5a"
+    codename: Hydra
+    releaseDate: 1996-06-30
+    support: 2002-01-01
+    eol: 2002-01-01
+    extendedSupport: false
+    latest: "6.50.480"
+    latestReleaseDate: 2005-10-07
+    link: https://support.microsoft.com/help/238621
+
+-   releaseCycle: "6.0"
+    releaseLabel: "6.0 SP3"
+    codename: SQL95
+    releaseDate: 1995-06-13
+    support: 1999-03-31
+    eol: 1999-03-31
+    extendedSupport: false
+    latest: "6.0.151"
+    latestReleaseDate: 2005-10-07
+    link: https://support.microsoft.com/help/152616
```

Comment:
The links for 7.0, 6.5, and 6.0 give me a 404.

Reply:
I know, they are from sqlserverversions but Microsoft has removed the KB articles :disappointed: 
I found betaarchive mentioned in PR description with a copy of the old articles but it's not the official source.
What's your opinion?
1. Leave it with the official links but gives 404
2. Replace with betaarchive
3. Remove with `link: null`

Reply:
Resolved with https://github.com/endoflife-date/endoflife.date/pull/2549/commits/56a8251e27c562832a7f033160ef21a2e753a83d